Reference_Manual.md

```diff
@@ -940,9 +944,123 @@ namespace hazelcast {
 
 Note that, this is very similar to `identified_data_serializer` implementation except the method signatures and the derived marker class of `portable_serializer` for the specialized serializer.
 
-`hz_serializer<Person>` specialization of `hz_serializer` should implement the above four methods, namely `get_factory_id`, `get_class_id`, `write_portable`, `read_portable`. In case that the object fields are non-public, you can always define struct `hz_serializer<Person>` as friend to your object class. You use the `portable_writer` class methods when serializing the object into binary bytes and you use the `portable_reader` class methods when de-serializing the bytes into the concrete object instance. 
+`hz_serializer<Person>` specialization of `hz_serializer` should implement the above four methods, namely `get_factory_id`, `get_class_id`, `write_portable`, `read_portable`. In case that the object fields are non-public, you can always define struct `hz_serializer<Person>` as friend to your object class. You use the `portable_writer` class methods when serializing the object into binary bytes and you use the `portable_reader` class methods when de-serializing the bytes into the concrete object instance.
 
-## 4.3. Custom Serialization
+## 4.3. compact_serializer Serialization
+As an enhancement to existing serialization methods, Hazelcast offers compact serialization, with the following main features.
+
+- Separates the schema from the data and stores it per type, not per object which results in less memory and bandwidth usage compared to other formats
```

Comment:
```suggestion
- Separates the schema from the data and stores it for each type instead of each object which results in less memory and bandwidth usage compared to other formats,
```

Reply:
Fixed.